### PR TITLE
!str #17277 remove akka.stream.javadsl.japi.WithVariance

### PR DESCRIPTION
**Pull request by rkuhn**
sadly we forgot to also move the boilerplate-generated FunctionN and
ProcedureN into akka-actor, so they are still in
akka.stream.javadsl.japi.

### CODE REVIEW

**Comment by rkuhn** [comment]
fixes #17277

**Comment by ktoso** [comment]
PLS BUILD

File: akka-samples/akka-docs-java-lambda/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <publishedAkkaVersion>2.3.9</publishedAkkaVersion>
+    <publishedAkkaVersion>2.3.10</publishedAkkaVersion>
```
Comment (patriknw) [inline akka-samples/akka-docs-java-lambda/pom.xml]: this is also in build.sbt in this project

**Comment by patriknw** [comment]
LGTM, after the build.sbt
sorry for forgetting the boilerplate

**Comment by rkuhn** [comment]
Well, that’s the beauty of teamwork: we share both the joy and the tears—in this case we all overlooked those functions.

**Comment by drewhk** [comment]
LGTM

**Comment by patriknw** [comment]
yes, I know, I'm still sorry

**Pull request merged**